src/appleseed-max-impl/appleseedoslplugin/oslshaderregistry.cpp

```diff
@@ -840,7 +842,7 @@ void OSLShaderRegistry::add_input_parameter(
             param_id,
             param_str.c_str(),
             TYPE_TEXMAP,
-            flag,
+            flag | is_deprecated,
```

Comment:
Maybe rename `flag` to `auto_labels_bit`?

Reply:
May be `flag_bits` because it's not just about auto_label?

Reply:
or it is about auto_labels here, so it makes sense for it to be auto_label_bit